broker/lib/controllers/ServiceFabrikApiController.js

```diff
@@ -32,6 +35,72 @@ const CloudControllerError = {
   }
 };
 
+
+function getResourceAnnotationStatus(resourceType, resourceId, guid, start_state, started_at) {
+  logger.info(`getResourceAnnotationStatus is called`);
+  return Promise.delay(CONST.EVENTMESH_POLLER_DELAY)
+    .then(() => eventmesh.server.getAnnotationState({
+      resourceType: resourceType,
+      resourceId: resourceId,
+      annotationName: 'backup',
+      annotationType: 'default',
```

Comment:
default is the operation `backup` type, it should be part of catalog and read from catalog instead of hardcoding.